src/code42cli/cmds/securitydata.py

```diff
@@ -25,15 +26,21 @@
 _HEADER_KEYS_MAP["name"] = "Name"
 _HEADER_KEYS_MAP["id"] = "Id"
 
+SEARCH_DEFAULT_HEADER = OrderedDict()
```

Comment:
Some of the fields can be removed since having so many fields doesn't fit in single terminal row. 

Added these fields based on the idea that,  users can specify these fields in search criteria ( advanced_query )

Reply:
The eventId is not particularly useful information and I would exclude it by default.
What order are the fields presented in? I think identifying information such as the file name and path should come first.

Reply:
The order is same as they are added in the dict.

Reply:
"FilePath"
 "FileName"
 "Type"
 "EventTimestamp"
 "FileCategory"
 "FileSize"
 "FileOwner"
 "MD5Checksum"
"SHA256Checksum"

Reply:
Just removed EventId and brought FilePath and FileName above.

Reply:
what do you think about having fileName be first?

Reply:
Changed.